supertokens_python/recipe/emailverification/__init__.py

```diff
@@ -13,26 +13,45 @@
 # under the License.
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Callable
+from typing import TYPE_CHECKING, Callable, Union, Optional, Any, Awaitable, Dict
 
 from . import exceptions as ex
 from . import utils
 from .emaildelivery import services as emaildelivery_services
-from .recipe import EmailVerificationRecipe
+from . import recipe
+from .interfaces import TypeGetEmailForUserIdFunction
+from .types import EmailTemplateVars, User
+from ...ingredients.emaildelivery.types import EmailDeliveryConfig
 
 InputOverrideConfig = utils.OverrideConfig
 ParentRecipeEmailVerificationConfig = utils.ParentRecipeEmailVerificationConfig
 exception = ex
 SMTPService = emaildelivery_services.SMTPService
+EmailVerificationClaim = recipe.EmailVerificationClaim
+EmailVerificationRecipe = recipe.EmailVerificationRecipe
 
 
 if TYPE_CHECKING:
     from supertokens_python.supertokens import AppInfo
 
     from ...recipe_module import RecipeModule
 
+from .utils import MODE_TYPE, OverrideConfig
+
 
 def init(
-    config: ParentRecipeEmailVerificationConfig,
+    mode: MODE_TYPE = "OPTIONAL",
```

Comment:
there is no default value to this. The user must pass some value themselves.

Reply:
Okay. Removed the default value.

Done.